internal/amf/gmm/handler.go

```diff
@@ -1720,7 +1737,7 @@ func sendServiceAccept(ctx ctxt.Context, ue *context.AmfUe, anType models.Access
 			if err != nil {
 				return fmt.Errorf("error sending initial context setup request: %v", err)
 			}
-			ue.GmmLog.Info("Sent NGAP initial context setup request")
+			ue.GmmLog.Info("Sent NGAP initial context setup request with context list", zap.Int("len", len(ctxList.List)))
```

Comment:
We already have a log that will come from the ngap code. Here however we may want to log what's nas specific and can't be infered from the NGAP piece. Ex. `"Sent service accept with xyz"`